chain/network/src/routing_table_actor.rs

```diff
@@ -0,0 +1,293 @@
+use std::collections::HashMap;
+
+use actix::dev::MessageResponse;
+use actix::{Actor, Handler, Message, SyncContext, System};
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use tracing::error;
+
+use near_performance_metrics_macros::perf;
+use near_primitives::network::PeerId;
+
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::ibf::{Ibf, IbfBox};
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::ibf_peer_set::IbfPeerSet;
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::ibf_set::IbfSet;
+use crate::routing::Edge;
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::routing::SimpleEdge;
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::routing::ValidIBFLevel;
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::routing::MIN_IBF_LEVEL;
+use crate::types::StopMsg;
+#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+use crate::types::{PartialSync, RoutingState, RoutingVersion2};
+
+/// Actor that maintains routing table information.
+/// TODO (PIOTR, #4859) Finish moving routing table computation to new thread.
```

Comment:
Should this be removed?

Reply:
@bowenwang1996 This is part 1 of the full change. In the part 1 we offload work of getting routing table to another thread.
In part 2 we will move `RoutingTable -  pub raw_graph: Graph,` to another thread.

So, this shouldn't be marked as done yet.